cli/vite.config.ts

```diff
@@ -2,6 +2,7 @@ import { defineConfig } from 'vite';
 import tsconfigPaths from 'vite-tsconfig-paths';
 
 export default defineConfig({
+  resolve: { alias: { src: '/src' } },
```

Comment:
Why is this needed?

Reply:
I followed `cli/README.md` and filed to build the project without this config (on Windows).

![image](https://github.com/immich-app/immich/assets/16451516/d296e163-6604-45e9-9c98-6738c05869e1)

Generally speaking, I think this change will improve the compatibility of dev configs.